_includes/scenarios/integrate/integration-example.md

```diff
@@ -1,151 +1,114 @@
-        ]
-      }
-    ]
-  }
-}
-```
-
-{% include rapidoc api_route_regex="^post /api/v2/Requests/Outgoing/Validate$" %}
-
-Even though the Requests are validated during the RelationshipTemplate creation you should not skip this step as it gives you additional information in case of validation errors.
+Next, we want to create a Relationship Template, that can be used by the App to send a Relationship Request to our Connector.
+The content of the template can be widely configured, but for simplicity we will use just two [RequestItemGroups]({% link _docs_integrate/data-model-overview.md %}#requestitemgroup) in our example.
+On the one hand, we want to share data with the App, namely the display name of our Connector we created in the previous step.
+For this, we use a [ShareAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#shareattributerequestitem).
+On the other hand, we use [ReadAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem)s to query information of the App.
+Let's assume the Connector needs to know the given name and surname of its contact to create a Relationship and, additionally, offers the option to specify an e-mail address for communication.
+
+| RequestItemGroups |                               |                               |
+| ----------------- | ----------------------------- | ----------------------------- |
+| `@type`           | `"RequestItemGroup"`          | `"RequestItemGroup"`          |
+| `title`           | `"Shared Attributes"`         | `"Requested Attributes"`      |
+| `items`           | `<ShareAttributeRequestItem>` | `<ReadAttributeRequestItems>` |
+| `mustBeAccepted`  | `true`                        | `true`                        |
+
+| ShareAttributeRequestItem |                                                        |
+| ------------------------- | ------------------------------------------------------ |
+| `@type`                   | `"ShareAttributeRequestItem"`                          |
+| `attribute`               | `<IdentityAttribute created in the previous step>`     |
+| `sourceAttributeId`       | `"<id of the attribute created in the previous step>"` |
+| `mustBeAccepted`          | `true`                                                 |
+
+| ReadAttributeRequestItems |                              |
+| ------------------------- | ---------------------------- | ---------------------------- | ---------------------------- |
+| `@type`                   | `"ReadAttributeRequestItem"` | `"ReadAttributeRequestItem"` | `"ReadAttributeRequestItem"` |
+| `query.@type`             | `"IdentityAttributeQuery"`   | `"IdentityAttributeQuery"`   | `"IdentityAttributeQuery"`   |
+| `query.valueType`         | `"GivenName"`                | `"Surname"`                  | `"EMailAddress"`             |
+| `mustBeAccepted`          | `true`                       | `true`                       | `false`                      |
```

Comment:
can you explain why these tables are better than the json of the old version? 

cc @stnmtz 

1. the reader of this guide HAS TO HAVE json knowledge anyway
2. with these tables the reader has to transform the tables to json, this is a massive overhead

Reply:
same for every table

Reply:
this is the scenario that should be possible even with very little knowledge. we may have to find an alternative here.

Reply:
this "scenario" is the main entrance to integrating into the enmeshed world. knowing how to read json should be the "very little knowledge" we should assume the user to have.

Reply:
IMO understanding how to transform these tables to JSON is **so much harder** than just knowing how to read JSON.